prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Absence {
+  id                  Int      @id @default(autoincrement())
+  lessonDate          DateTime
+  subject             Subject
+  lessonPlan          String?
+  reasonOfAbsence     String
+  numberOfStudents    Int
```

Comment:
```suggestion
  numberOfStudents    Int      @default(0)
```

Reply:
I think this was mentioned in one of the documents that they wanted # of students? Not sure

Reply:
I don't think it was, if you could provide me with a link or something that would be great.

I think it makes some sense (like a teacher might not want to attend a class if there are too many students) but I don't think this is a necessary feature at the moment, since our client did not indicate interest.